skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp

```diff
@@ -90,7 +91,7 @@ Object::PropInfo* ActivePexInstance::GetProperty(
   uint8_t flag)
 {
   if (!scriptInstance.IsValid())
-    return nullptr;
+   throw std::runtime_error("Error");
```

Comment:
Very unclear error. Please try to explain something here or at least make something that will clearly send us to this specific place if an error occurs. For this line, that could be something like `"Papyrus script instance is not valid"`

Reply:
Hmm by the way this shouldn't have been replaced, it just returns a nullptr for a valid case (as far as I can tell, I don't remember exactly)